dev-python/traits/traits-4.5.0.ebuild

```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2016 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
```

Comment:
This is a stable ebuild, don't do `EAPI` bumps in place. Just restore this one to its old state.